src/types.ts

```diff
@@ -106,3 +116,7 @@ export type RunnerState = {
   bitbucketBaseUrl: string;
   permissionsMessage: string;
 };
+
+export type MergeOptions = {
```

Comment:
Where does this type get used?

Reply:
Here: https://github.com/atlassian/landkid/pull/109/files#diff-31195c1d7289295f28e7902fd02c31cad4122633ef305054421ae836ce368f63R17